### PR TITLE
testing git

### PR DESCRIPTION
**Pull request by ghost**
Test

### CODE REVIEW

File: server/server.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ app.post('/search', (req: any, res: any) => {
     results: topTenSearches
   }));
 });
-
+//async and await is so fun ://
```
Comment (RafaelPiloto10) [inline server/server.ts]: This is an example change request where I am asking you to remove this comment

**Review by RafaelPiloto10 — CHANGES_REQUESTED** [review]

File: server/server.ts (hunk 1 above)
Comment (RafaelPiloto10) [reply server/server.ts]: Hit resolve comment below and I will approve

Comment (ghost) [reply server/server.ts]: So lets say I made the change and re-pushed into this branch, how would I make the request again, the request button is not longer here

Comment (ghost) [reply server/server.ts]: Nevermind, refresh page is op

**Review by RafaelPiloto10 — APPROVED** [review]
Now you can merge! :)

**Pull request merged**